code/modules/mob/living/basic/slime/slime_type.dm

```diff
@@ -1,6 +1,8 @@
 /datum/slime_type
 	///Our slime's colour as text. Used by both description, and icon
 	var/colour
+	///Whether the slime icons should be semi-transparent
+	var/transparent
```

Comment:
```suggestion
	var/transparent = FALSE
```

so we can "type" it as a boolean, rather than have a weird `null`/`TRUE` dichotomy